app-backend/db/src/test/scala/com/azavea/rf/database/AnnotationProjectDaoSpec.scala

```diff
@@ -332,4 +335,116 @@ class AnnotationProjectDaoSpec
       )
     }
   }
+
+  test("list projects belonging to a campaign") {
+    check {
+      forAll(
+        (
+            userCreate: User.Create,
+            annotationProjectCreates: List[AnnotationProject.Create],
+            taskFeaturesCreate: Task.TaskFeatureCollectionCreate,
+            campaignCreate: Campaign.Create
+        ) => {
+          val pageSize = 20
+          val pageRequest = PageRequest(0, pageSize, Map.empty)
+
+          val listIO = for {
+            user <- UserDao.create(userCreate)
+            insertedCampaign <- CampaignDao
+              .insertCampaign(campaignCreate, user)
+            insertedProjects <- annotationProjectCreates
+              .take(pageSize) traverse { toInsert =>
+              AnnotationProjectDao
+                .insert(
+                  toInsert.copy(campaignId = Some(insertedCampaign.id)),
+                  user
+                )
+            }
+            _ <- insertedProjects traverse { project =>
+              TaskDao.insertTasks(
+                fixupTaskFeaturesCollection(
+                  taskFeaturesCreate,
+                  project,
+                  Some(TaskStatus.Unlabeled)
+                ),
+                user
+              )
+            }
+            listed <- AnnotationProjectDao
+              .listProjects(
+                pageRequest,
+                AnnotationProjectQueryParameters(
+                  campaignId = Some(insertedCampaign.id)
+                ),
+                user
+              )
+          } yield (listed, insertedProjects)
+
+          val (listedProjects, dbProjects) = listIO.transact(xa).unsafeRunSync
+
+          val expectedIds = (dbProjects.take(pageSize) map { _.id }).toSet
+
+          assert(
+            expectedIds == (listedProjects.results map { _.id }).toSet,
+            "Listed projects are those expected from projects belonging to an inserted campaign"
+          )
+
+          true
+        }
+      )
+    }
+  }
+
+  test("list projects by capture timestamp") {
+    check {
+      forAll(
+        (
+            userCreate: User.Create,
+            annotationProjectCreates: List[AnnotationProject.Create],
+            taskFeaturesCreate: Task.TaskFeatureCollectionCreate
+        ) => {
+          val pageSize = 20
+          val pageRequest = PageRequest(0, pageSize, Map.empty)
+          val capturedAt = Timestamp.from(Instant.now());
+
+          val listIO = for {
```

Comment:
Similar to above, this test doesn't prove that the filtering works without a second annotation project create list or some of the annotation projects getting a different `capturedAt` value